tools/tune/multiband_drc/example_multiband_drc.m

```diff
@@ -139,7 +139,7 @@ function export_multiband_drc(prm)
 				      endian, 4);
 
 tplg_write(tplg1_fn, blob8, "MULTIBAND_DRC");
-tplg2_write(tplg2_fn, blob8_ipc4, "multiband_drc_config", 'Exported Control Bytes');
+tplg2_write(tplg2_fn, blob8_ipc4, "multiband_drc_config", "Exported with script example_multiband_drc.m");
```

Comment:
btw, next steps would be to include the command line and tool version used to generate the data so that any developer can reproduce.

Reply:
Yep, we can use more comment estate than this with a bit more changes.